qemu/tests/block_resize.py

```diff
@@ -57,20 +56,12 @@ def compare_block_size(session, block_cmd, block_pattern):
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     timeout = float(params.get("login_timeout", 240))
-    session = vm.wait_for_login(timeout=timeout)
+    driver_name = params["driver_name"]
```

Comment:
better to use params.get("driver_name"), as i cannot found driver_name for linux guest in configure file.